packages/synapse-react-client/src/components/Authentication/AuthenticationMethodSelection.tsx

```diff
@@ -44,34 +47,16 @@ export default function AuthenticationMethodSelection(
 
   return (
     <Box>
-      <Button
```

Comment:
Refactor these buttons into a LoginMethodButton component, so can add other login method buttons (e.g. NIH, ORCID) in the future